data/mods/MindOverMatter/recipes/practice/electrokinesis_practice.json

```diff
@@ -0,0 +1,659 @@
+      "practice_electrokinetic_personal_battery",
+      "practice_electrokinetic_paralysis",
+      "practice_electrokinetic_lightning_bolt",
+      "practice_electrokinetic_speed_boost",
+      "practice_electrokinetic_kill_robot",
+      "practice_electrokinetic_lightning_aura",
+      "practice_electrokinetic_lightning_blast",
+      "practice_electrokinetic_revive"
+    ],
+    "difficulty": 1
+  },
+  {
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "name": "contemplation: spark sight",
+    "id": "practice_electrokinetic_see_electric",
+    "description": "Contemplate your powers and improve your ability to detect electrical creatures or robots.",
+    "category": "CC_*",
+    "subcategory": "CSC_*_NESTED",
+    "skill_used": "metaphysics",
+    "difficulty": 0,
+    "time": "30 m",
+    "autolearn": false,
+    "tools": [ [ [ "matrix_crystal_drained", -1 ] ] ],
+    "flags": [ "SECRET", "BLIND_HARD" ],
+    "result_eocs": [
+      {
+        "id": "EOC_PRACTICE_ELECTROKIN_SPARK_SIGHT",
+        "condition": {
+          "and": [
+            { "math": [ "u_val('spell_level', 'spell: electrokinetic_see_electric')", ">=", "1" ] },
+            {
+              "math": [ "u_val('spell_exp', 'spell: electrokinetic_see_electric')", "<=", "(difficulty_one_contemplation(1))" ]
+            }
+          ]
+        },
+        "effect": [
+          { "u_message": "You spend some time meditating and contemplating your powers and emerge with new knowledge." },
+          {
+            "math": [ "u_val('spell_exp', 'spell: electrokinetic_see_electric')", "+=", "(contemplation_factor(1))" ]
+          },
+          { "math": [ "u_val('vitamin', 'name:vitamin_psionic_drain')", "+=", "rng( 0,2 )" ] },
+          { "math": [ "u_val('stored_kcal')", "-=", "psionics_contemplation_kcal_cost(1)" ] },
+          { "run_eocs": "EOC_PSI_PRACTICE_FOCUS_MOD" }
+        ],
+        "false_effect": {
+          "run_eocs": [
+            {
+              "id": "EOC_PRACTICE_ELECTROKIN_SPARK_SIGHT_FALSE",
+              "condition": { "math": [ "u_val('spell_level', 'spell: electrokinetic_see_electric')", ">=", "1" ] },
+              "effect": [
+                { "u_message": "Your knowledge of your powers is so deep that mere contemplation is of no further use to you." },
+                { "run_eocs": "EOC_PSI_PRACTICE_FOCUS_MOD" }
+              ],
+              "false_effect": [
+                { "u_message": "Without even a basic understanding of the power, your meditation is nothing but idle musings." },
+                { "run_eocs": "EOC_PSI_PRACTICE_FOCUS_MOD" }
+              ]
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "name": "contemplation: shock touch",
+    "id": "practice_electrokinetic_shock_touch",
+    "description": "Contemplate your powers and improve your ability build up a charge in your hand and unleash it.",
```

Comment:
```suggestion
    "description": "Contemplate your powers and improve your ability to build up a charge in your hand and unleash it.",
```

Reply:
Yeah (and thank you for reminding me I need to include the abilities/limitations in the docs).  I've been trying to avoid powers like that for two reason:

1) I stuck with the idea of things electricity can "naturally" do. So you can drain electricity from batteries (because batteries discharge) and give living beings seizures (because that happens) but not drain the bio-energy from someone's nervous system (because that's not something that would ever happen under any natural circumstances). 

2) Feral psions can do all the same things that the player can do. One of the main reasons I haven't included anything like that, or telekinetically closing someone's carotid artery, teleporting their heart out of their chest, etc., is that then there would be no reason for ferals not to be able to do it. (The other reason is that insta-kills that can't be defended against aren't really interesting on either side of the table).